macos/package_files/3.10.0/preinstall.sh

```diff
@@ -17,6 +17,17 @@ else
     launchctl setenv WAZUH_PKG_UPGRADE true
 fi
 
+if [ `launchctl getenv WAZUH_PKG_UPGRADE` == true ]; then
+    mkdir -p ${DIR}/config_files/
+    cp -r $DIR/etc/{ossec.conf,client.keys,local_internal_options.conf,shared} $DIR/config_files/
```

Comment:
Change `$DIR` to `${DIR}`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/3a2266309751f5e616ee1e44cf19e96350617fa9 and https://github.com/wazuh/wazuh-packages/pull/221/commits/1cb5ebf17dba1bb08ef9d4b8eee71a706945d8ac

Reply:
The line 22 isn't changed yet.